Xamarin.Essentials/Email/Email.mac.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Email
```

Comment:
If you create a "mailto:" url and use `OpenUrl` then you can support most of these features.